README.md

```diff
@@ -1 +1,69 @@
-# Calculator-React
\ No newline at end of file
+# Calculator-React
```

Comment:
**Let's make your repo looks professional by applying all the 3 basic rules, kindly update your `README` file and remove unnecessary content**.
Checkout out the template for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright

![image](https://user-images.githubusercontent.com/31897434/92685488-b4c45680-f32f-11ea-9bea-222a21c096a6.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/